### DISCUSSION TITLE
talk to server... ERROR 429 Too Many Requests

### DISCUSSION THREAD

**Discussion by Mbertao** [wazuh/wazuh#17859] [Q&A]
Hello everyone, I would like some help with this error.

filebeat test output
elasticsearch: https://X.x.x.x:9200...
  parse url... OK
  connection...
    parse host... OK
    dns lookup... OK
    addresses: x.x.x.x
    dial up... OK
  TLS...
    security: server's certificate chain verification is enabled
    handshake... OK
    TLS version: TLSv1.3
    dial up... OK
  talk to server... ERROR 429 Too Many Requests: {"error":{"root_cause":[{"type":"circuit_breaking_exception","reason":"[parent] Data too large, data for [<http_request>] would be [1036279800/988.2mb], which is larger than the limit of [1020054732/972.7mb], real usage: [1036279800/988.2mb], new bytes reserved: [0/0b], usages [request=0/0b, fielddata=0/0b, in_flight_requests=0/0b, accounting=99668824/95mb]","bytes_wanted":1036279800,"bytes_limit":1020054732,"durability":"PERMANENT"}],"type":"circuit_breaking_exception","reason":"[parent] Data too large, data for [<http_request>] would be [1036279800/988.2mb], which is larger than the limit of [1020054732/972.7mb], real usage: [1036279800/988.2mb], new bytes reserved: [0/0b], usages [request=0/0b, fielddata=0/0b, in_flight_requests=0/0b, accounting=99668824/95mb]","bytes_wanted":1036279800,"bytes_limit":1020054732,"durability":"PERMANENT"},"status":429}

**Comment by Mbertao** [comment]
Problem OK

**Reply by DanielK37** [reply]
Hi Mbertao,

I have exactly the same problem at the moment. I suddenly cannot connect to the Web GUI anymore and have the same filebeat status. I'm really happy for you, that you solved it but would you mind sharing the solution with me/us?

Thank you very much.

**Reply by DanielK37** [reply]
Don't worry. Obviously it is fixed now. It looks to have been a general problem of JVM exhaustion. I have increased the JVM values from 1g to 4g and now everything works again. -> /etc/wazuh-indexer/jvm.options
